INSTALL.rst

```diff
@@ -330,6 +330,13 @@ openPMD API
 """""""""""
 - 0.12.0+ (bare minimum) / 0.13.0+ (for streaming IO)
 - *Spack*: ``spack install openpmd-api``
+- For usage in PIConGPU, the openPMD API must have been built either with support for ADIOS2 or HDF5.
```

Comment:
I think this may be mistakenly understood that only one of those is supported. I think it can be rewritten as 
`For usage in PIConGPU, the openPMD API must have been built with support for ADIOS2 or HDF5 (or both)`